Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine AS builder
+
+WORKDIR /go/src/
```

Comment:
there's no reason to put this into the `/go/src` and is getting rather confusing when we're importing it in the runtime image from a `src` directory

Reply:
I agree with the docker ignore, but regarding the workdir if you dont specify it you will end up by adding the files to /go and then you cannot compile it because its the same path as $GOPATH. This will produce the error $GOPATH/go.mod exists but should not

Reply:
I was more thinking of `workdir /sshit`

Reply:
with that workdir you will end up with COPY --from=builder /sshit/shhgit . that also look weird for me... I would prefer to have /go/src since this is actually source code that you are using to build a binary

Reply:
mhhh
what we could do to have it in `src` and still not copy a binary from `src` would be to work with `-o` when building.
What do you think?